build.gradle

```diff
@@ -53,6 +53,7 @@ dependencies {
 
     implementation "org.apache.kafka:kafka_$scala_version:$kafka_version"
     implementation "com.github.ben-manes.caffeine:caffeine:$caffeine_version"
+    implementation 'org.slf4j:slf4j-api:1.7.32'
```

Comment:
Why do you need to introduce this dependency.

Reply:
Had to add this dependency because the one we had on following versions of Java and Kafka stopped working. Also now have made a change so it works dynamically in the same way the other do.